react-client-app/src/api/values.js

```diff
@@ -197,8 +199,8 @@ const saveMappingRules = async (scan_report_concept,scan_report_value,table) =>
         'drug_exposure':['drug_exposure_start_datetime','drug_exposure_end_datetime'],
 	'procedure_occurrence':['procedure_datetime'],
 	'specimen':['specimen_datetime']
-        }
-    const destination_field = await cachedOmopFunction(fields,domain+"_source_concept_id")
+        }      
+    const destination_field = await cachedOmopFunction(fields,domain+"_concept_id")
```

Comment:
as above, unsafe to assume domain+"_source_concept_id" exists for all omop tables... it does for concept_id.. i think